recipe/pyproject.patch

```diff
@@ -0,0 +1,56 @@
+--- /dev/null
++++ pyproject.toml
+@@ -0,0 +1,53 @@
++# This pyproject.toml file is used to build the conda package only.
++
++[build-system]
++requires = ["setuptools"]
++build-backend = "setuptools.build_meta"
++
++[project]
++name = "rdkit"
++description = "A collection of chemoinformatics and machine-learning software written in C++ and Python"
```

Comment:
```suggestion
+description = "A collection of cheminformatics and machine-learning software written in C++ and Python"
```

Reply:
Thanks, I should have been more careful. Most of the content is coming from https://github.com/kuelumbus/rdkit-pypi/blob/a00434d6a86e1845d58da374af4fdae94828eda7/setup.py

Reply:
> Thanks, I should have been more careful. Most of the content is coming from https://github.com/kuelumbus/rdkit-pypi/blob/a00434d6a86e1845d58da374af4fdae94828eda7/setup.py

yeah, I figured. I've just never done a PR review there in order to fix that horrible typo. ;-)